resolwe_bio/processes/reads_processing/trimmomatic.yml

```diff
@@ -11,53 +11,65 @@
     expression-engine: jinja
     executor:
       docker:
-        image: resolwebio/legacy:1.0.0
-  data_name: '{{ reads.fastq.0.file|default("?") }}'
-  version: 0.1.3
+        image: resolwebio/rnaseq:2.1.1
+  data_name: '{{ reads|sample_name|default("?") }}'
+  version: 0.1.4
```

Comment:
This is a braking change that will necessitate also the update of the workflows and configurations. Bump the first number and add braking change notice to the Changelog

Reply:
I have changed to version 1.0.0